index.html

```diff
@@ -0,0 +1,147 @@
+              <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+            <li class="nav-item">
+              <a class="nav-link"  href="#">News</a>
+            </li>         
+            <li class="nav-item GE">
+                <a class="nav-link" id="ge" href="#">Campaign</a>
+              </li>    
+          </ul>
+          </div>
+    </navbar>
+   <!--Header-->
+   <header>
+    <h1 class="heading1">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="parag">Welcome to our global annual summit that will be held virtually for ease of accessibility at your comfort.
+        For those who want to attend in person, it will be held at Uhuru Park, Nairobi,Kenya. </p>
+    <!-- Today's date -->
+    <h3 class="date"></h3>
+    <p class="sub">@ National Museum of Korea, Art Center and more</p>
+   </header>
+
+   <!-- Main program section -->
+   <section class="main-program">
+    <h3 class="heading3">Main Program</h3>
+    <hr class="hr">
+    <div class="grid-section">
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/save-water.png" alt="save-water"/>
+                <p class="first-p">Water</p>
+                <p class="second-p">We will discuss the benefits of avoiding water wastage.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/energy.png" alt="clean-energy"/>
+                <p class="first-p">Energy</p>
+                <p class="second-p">Exploration of energies that do not pollute environment such as biogas.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/plant.png" alt="conserve plants"/>
+                <p class="first-p">Forest</p>
+                <p class="second-p">Planting trees is very important in conservation of animal habitat and others that will be discussed.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/light.png" alt="Knowledge"/>
+                <p class="first-p">Knowledge</p>
+                <p class="second-p">Importance of teaching our young ones the benefit of conserving environment.</p>
+            </div>
+        </div>
+        <div class="grid-item">
+            <div class="item1">
+                <img src="assets/recycle.png" alt="Recycle"/>
+                <p class="first-p">Recycle</p>
+                <p class="second-p">Plastic waste contribute to a huge percent of sea animals suffocation.</p>
+            </div>
+        </div>
+        <div class="grid-item2">
+            <h3 class="join-p">Join Green Earth Global Summit</h3>
+        </div>
+
+    </div>
+    <h3 class="heading31">SEE THE WHOLE PROGRAM</h3>
+    <hr class="hrprog">
+   </section>
```

Comment:
- [ ] Kindly note that you have to stick to the design guidelines, kindly address the following:

   -  Add the pattern background included in the design.
   -  Make sure to add the hover effect on the cards as shown in the design.
   -  Make sure to use the basic color swatch: #272A31, #EC5242, #D3D3D3
  
| Your design | Actual design |
| ------------- | ------------- |
| ![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/102612396/f0773e4c-d357-430f-956b-90b770011bb1)  | ![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/102612396/e6df8c28-1f82-46e2-94f0-27f4068c3979) |